tests/common.py

```diff
@@ -934,7 +934,7 @@ def __init__(
             kwargs["state"] = state
         super().__init__(**kwargs)
         if reason is not None:
-            self.reason = reason
+            object.__setattr__(self, "reason", reason)
```

Comment:
We have to do this to avoid the check thats coming in #110023  

We don't need to worry about the internal state in tests here since its only used for log messages